.github/workflows/build.yml

```diff
@@ -13,7 +13,22 @@ concurrency:
 jobs:
   build:
     runs-on: ubuntu-latest
-    timeout-minutes: 15
+    timeout-minutes: 20
+    strategy:
+      matrix:
+        # INFO: This is only a workaround to avoid timeout issues with the CI;
+        # whether it becomes a definitive solution or not will require a
+        # separate discussion with Oscar.
+        cycle:
```

Comment:
@oscar-izval Please let me know if this appears to be a reasonable solution for the time being, as I believe it is important to address the issue reported in #53